tests/functional/multi-modules/module3.js

```diff
@@ -1,6 +1,6 @@
-const Werelogs = require('werelogs').Logger;
+const Werelogs = require('werelogs').Werelogs;
 
-const log = new Werelogs('test-mod3');
+const log = new Werelogs('test-mod3', { logLevel: 'info' }).Logger;
```

Comment:
That looks like what I intended to provide; but with your change, does new Werelogs().Logger even work ? I was under the understanding that Logger is no longer a constructor function but an object instance already, and I expect this not to work.